uint8 P = 0
uint8 PI = 1
uint8 PD = 2
uint8 PID = 3
uint8 PESSEN_INTEGRAL = 4
uint8 SOME_OVERSHOOT = 5
uint8 NO_OVERSHOOT = 6
uint8 method_type